sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/CallingServerClient.java

```diff
@@ -154,6 +154,20 @@ public CallConnection joinCall(
         return callingServerAsyncClient.joinWithResponseInternal(callLocator, source, joinCallOptions, context).block();
     }
 
+    /**
+     * Answer a call
+     *
+     * @param incomingCallContext The incoming call context.
+     * @throws CallingServerErrorException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return CallConnection for a successful answer request.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public CallConnection answerCall(
```

Comment:
we need an answercallwithresponse too, correct?

Reply:
added